api/api_controller_test.go

```diff
@@ -486,7 +485,7 @@ func TestHandler_ListBranchesHandler(t *testing.T) {
 		if err != nil {
 			t.Fatalf("unexpected error listing branches: %s", err)
 		}
-		const expectedBranchesLen = 2 // branch creation and import branch
+		const expectedBranchesLen = 1
```

Comment:
This difference from the change at l.235 (-1 here, -2 there) makes me wonder: either the old way was strange or the new way is strange.  I hope we're sure that the old way is strange :-)

Reply:
The old way is definitely strange.